components/stacks-network/src/orchestrator.rs

```diff
@@ -38,6 +39,29 @@ pub struct DevnetOrchestrator {
     subnet_node_container_id: Option<String>,
     subnet_api_container_id: Option<String>,
     docker_client: Option<Docker>,
+    services_map_hosts: Option<ServicesMapHosts>,
+}
+
+// pub enum DevnetServices {
+//     BitcoinNode,
+//     StacksNode,
+//     StacksApi,
+//     StacksExplorer,
+//     BitcoinExplorer,
+//     SubnetNode,
+//     SubnetApi,
+// }
```

Comment:
Is this comment left there on purpose?

Reply:
Yeah, I started using it, then rolled back. I think it could be nice for future refactoring, but I don't have the bandwidth right now :/